qutebrowser/completion/models/miscmodels.py

```diff
@@ -103,20 +103,28 @@ def session(*, info=None):
     return model
 
 
-def _tabs(*, win_id_filter=lambda _win_id: True, add_win_id=True):
+def _tabs(*, win_id_filter=lambda _win_id: True, add_win_id=True, cur_win_id=0):
     """Helper to get the completion model for tabs/other_tabs.
 
     Args:
         win_id_filter: A filter function for window IDs to include.
                        Should return True for all included windows.
         add_win_id: Whether to add the window ID to the completion items.
+        cur_win_id: Window ID to be passed from info.win_id
     """
     def delete_tab(data):
         """Close the selected tab."""
-        win_id, tab_index = data[0].split('/')
+        unpacked_data = data[0].split('/')
+
+        if len(unpacked_data) == 1:
+            win_id = cur_win_id
+        else:
+            win_id = unpacked_data[0]
```

Comment:
I think this would be a bit clearer as something like:

```suggestion
        if cur_win_id is None: 
            win_id, tab_index = data[0].split('/')
        else:
            win_id = current_win_id
            tab_index = data[0]
 ```

or so, with the default value changed accordingly. That way, accidental misuse of the function is prevented too.